packages/uix-guest/src/guest.ts

```diff
@@ -196,7 +196,7 @@ export class Guest<
             this.hostConnection.getRemoteApi().invokeHostMethod,
             address
           ),
-          10000
+          20000
```

Comment:
Do we really need to increase timeout for method execution? I just believed we need it only for extension loading

Reply:
My initial thinking was slow connections would impact method calls - but now I think when we call these methods, code is already loaded (otherwise extension wouldn't operate at all).
So, I will remove this timeout change but keep others.